android/src/main/java/com/swmansion/reanimated/NodesManager.java

```diff
@@ -228,7 +230,8 @@ private void onAnimationFrame(long frameTimeNanos) {
     currentFrameTimeMs = frameTimeNanos / 1000000.;
 
     while (!mEventQueue.isEmpty()) {
-      handleEvent(mEventQueue.poll());
+      CopiedEvent copiedEvent = mEventQueue.poll();
+      handleEvent(copiedEvent.getTargetTag(), copiedEvent.getEventName(), copiedEvent.getPayload());
```

Comment:
You should call `copiedEvent.onDispose()` here to not break the Event contract.

Reply:
CopiedEvent is just a plain object and doesn't have onDispose method.

Reply:
I see, my bad.